core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -1097,7 +1098,12 @@ public SystemSessionProperties(
                 durationProperty(CLOSE_IDLE_WRITERS_TRIGGER_DURATION,
                         "The duration after which the writer operator tries to close the idle writers",
                         new Duration(5, SECONDS),
-                        true));
+                        true),
+                booleanProperty(
+                        ALLOW_UNSAFE_PUSHDOWN,
+                        "Allow pushing down expressions that may fail for some inputs",
+                        optimizerConfig.isUnsafePushdownAllowed(),
+                        false));
```

Comment:
This should probably be hidden or at least state in the description that it is teporary measure to be removed.

Reply:
Good point. I'll hide it.

I think the name itself conveys that it's not something to do lightly.